src/rust/ide/view/src/project.rs

```diff
@@ -57,8 +58,34 @@ impl Model {
     }
 
     fn set_style(&self, is_light:bool) {
-        if is_light { self.app.themes.set_enabled(&["dark"])  }
-        else        { self.app.themes.set_enabled(&["light"]) }
+        if is_light { self.set_dark_style()  }
+        else        { self.set_light_style() }
+    }
```

Comment:
What do oyu need this function for if you have now the enum `theme::Theme`? Just merge this function with `set_style_by_theme` which has a much better API than by providing a "bool"